docker/buildtools/Dockerfile

```diff
@@ -0,0 +1,84 @@
+# This Dockerfile provides all build tools that FOSSA CLI invokes. It's useful
+# for developing the FOSSA CLI on machines that may not have all the necessary
+# build tools.
+FROM fossa/fossa-cli:base
+
+# Workaround for add-apt-repository (see
+# https://github.com/oerdnj/deb.sury.org/issues/56).
+ENV LC_ALL=C.UTF-8 DEBIAN_FRONTEND=noninteractive 
+
+# Install build tool requirements.
+RUN sudo apt-get install -y \
+    apt-transport-https build-essential libssl-dev software-properties-common
+
+# Install JVM version manager.
+RUN sudo apt-get install -y zip unzip && \
+    curl https://get.sdkman.io | bash
+
+# Install JVM runtimes and build tools.
+RUN ["/bin/bash", "-c", "\
+    source $HOME/.sdkman/bin/sdkman-init.sh && \
+    # Install Java runtime and build tools
+    sdk install java && \
+    sdk install maven && \
+    sdk install gradle && \
+    sdk install ant && \
+    # Install Scala runtime and build tools
+    sdk install scala && \
+    sdk install sbt \
+    "]
+RUN mkdir -p $HOME/.sbt/1.0/plugins && \
+    echo 'addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.9.0")' > $HOME/.sbt/1.0/plugins/plugins.sbt
+
+# Install Android SDK.
+RUN wget https://dl.google.com/android/repository/sdk-tools-linux-3859397.zip -O /tmp/sdk-tools-linux.zip && \
+    mkdir -p $HOME/Android/Sdk && \
+    unzip /tmp/sdk-tools-linux.zip -d $HOME/Android/Sdk
+ENV PATH=$PATH:/home/fossa/Android/Sdk/tools/bin ANDROID_HOME=/home/fossa/Android/Sdk
+
+# Install additional Go build tools.
+RUN curl https://glide.sh/get | bash && \
+    go get -u -v github.com/tools/godep && \
+    go get -u -v github.com/kardianos/govendor && \
+    go get -u -v github.com/LK4D4/vndr && \
+    go get -u -v github.com/sparrc/gdm
```

Comment:
minus well add in `delve` for debugging here 
`go get -u github.com/derekparker/delve/cmd/dlv`

Reply:
I've added this to the `base` image instead.